datadog/getmetrics_datadog.py

```diff
@@ -415,13 +415,18 @@ def set_logger_config(level):
             for sub_metric_list in chunked_metric_list:
                 for sub_host_list in chunked_host_list:
                     # get metric data from datadog every SAMPLING_INTERVAL
+                    count = 0
                     for _ in xrange(ATTEMPTS):
```

Comment:
Just track the index (change _ to a variable name) and later check if index == ATTEMPTS on exception